clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -157,13 +158,23 @@
         </xs:annotation>
       </xs:element>
     </xs:sequence>
-    <xs:attribute name="auto-create" type="xs:boolean" use="optional" default="false">
+    <xs:attribute name="auto-create" type="xs:boolean" use="optional">
       <xs:annotation>
-        <xs:documentation> xml:lang="en">
+        <xs:documentation xml:lang="en">
           True if server side components should be automatically created if they are absent.
```

Comment:
It may be worth noting here that using both the `auto-create` and `client-mode` attributes is an error.